apps/src/applab/designMode.js

```diff
@@ -882,6 +949,16 @@ designMode.configureDragAndDrop = function () {
   // element tray to the play space.
   $('#visualization').droppable({
     accept: '.new-design-element',
+    activate: function (event, ui) {
+      // Turn off clipping in app space so we can drag the element
+      // into and out of it
+      setAppSpaceClipping(false);
```

Comment:
I tried to remove and it makes the animation a little odd when an element is dropped partially off screen. See below, so I decided to keep it.

#### Keeping this
![drag-in-before](https://cloud.githubusercontent.com/assets/1885340/15696565/a15f7aa0-277f-11e6-89c1-f55a392f487c.gif)

#### Removing this
![drag-in-after](https://cloud.githubusercontent.com/assets/1885340/15696566/aa5a46d0-277f-11e6-8184-3c058a2e50a8.gif)